### ISSUE TITLE
Suggested topic:  community archetypes

### ISSUE THREAD

**Issue by moxious** [fossified/podcast#4]
So ideally you'd get someone like Visakanv Veerswamy who would probably be a banger, but probably others could speak to this as well.

In FOSS we might say we have a couple of different archetypes of people who behave differently but all contribute something of value.

* Community managers do a lot of logistics stuff.  They're triagers, event setter-uppers, make the trains run on time
* Content people.  These could be the workhorse PR submitters, the talk-givers.  They have outsized impact on a community's voice because they're the most visible.  Content could be code, docs, presentations, whatever really.
* Switchboard operators [walk around and talk to a lot of people](https://twitter.com/visakanv/status/1368572047278759940?s=20) and act as social connectors.  You know you're talking to one when you ask a question and their answer is always "You should talk to...."
* Naturally, there's an archetype of BDFL in FOSS, people know that one pretty well.  Your Guido van Rossums & Linus Torvalds
* Lurkers.  They'll thumbs up your issue but never speak, but even they contribute community signal (this issue has 200 emoji reacts, that one has 3)

FOSS communities have social infrastructure in addition to technical infrastructure, in short.

**Comment by hesa** [comment]
Community manager sounds a bit like a role in a company (FOSS) project.  Lurker sounds more like a role in a community project. I figure you want all characters from all kinds of FOSS projects?

So, listing these and discussing them, possibly with a guest representing one/some of the characters. Perhaps inviting a company (who contributes a lot to FOSS) and have them explain and discuss their engagement.
* What is the need/goals/purpose of this role?
* Who is suited for this role?

Is the above close to what you had in mind?

BTW, another character we've all seen (and perhaps even are or have been): https://the.webm.ink/recognising-reputation-vampires

**Comment by moxious** [comment]
yes, that's broadly what I had in mind.  Adding negative archetypes of "problems typically seen in FOSS communities" would be an interesting addition as well.   The root of it though is that FOSS takes all kinds of different people, and their skills tend to cluster into these types of archetypes.  Understanding that is useful for thinking through what's wrong with a community and optionally how it can be improved.

community manager can be a role in a company project, but my point with that label is that productive, helpful community members not working for any company often _behave that way_ even if they're not employed for that purpose.
